test/Microsoft.AspNetCore.SignalR.Tests/EndToEndTests.cs

```diff
@@ -28,15 +29,19 @@ public class EndToEndTestsCollection : ICollectionFixture<ServerFixture>
     [Collection(EndToEndTestsCollection.Name)]
     public class EndToEndTests
     {
+        private readonly ITestOutputHelper _output;
+
         private readonly ServerFixture _serverFixture;
 
-        public EndToEndTests(ServerFixture serverFixture)
+        public EndToEndTests(ServerFixture serverFixture, ITestOutputHelper output)
```

Comment:
Yes. @BrennanConroy looked at the server logs for [the flaky test failure](https://travis-ci.org/aspnet/SignalR/jobs/209849075) and recommended turning on logs on the client.